src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/EventListener.kt

```diff
@@ -10,14 +10,16 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.future.future
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.listeners.EventFeatureSwitch
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.listeners.SQSMessage
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.prisonperson.physicalattributes.SynchronisationService
 import java.util.concurrent.CompletableFuture
 
-@Service
-class PrisonPersonEventListener(
-  private val prisonPersonService: PrisonPersonSynchronisationService,
+@Service("prisonPersonEventListener")
+class EventListener(
+  @Qualifier("physicalAttributesSynchronisationService") private val physicalAttributesSyncService: SynchronisationService,
```

Comment:
I've removed the `PrisonPerson` prefix from all classes as they make it hard on the eyes. So there is lots of renaming / qualifying beans so they don't clash with other packages.

Reply:
Although if everyone did that Command-O would be pretty useless in IntelliJ right?

Reply:
I'm going to have lots of these for my various migrations. Let's see how it goes. I despise prefixing class names with the package name 😆